src/domUpdates.js

```diff
@@ -62,10 +62,26 @@ const renderGrid = () => {
   recipeGrid.innerHTML = createGridHTML(gridData);
 }
 
+const showRecipe = (recipeCard) => {
+  const recipeCardName = recipeCard.lastElementChild.textContent;
```

Comment:
I feel lastElementChild could be a problem IF the html warranted some change and somehow, the required text wouldn't be the last element?
What do you feel about 
event.target.closest(".individual-recipe").querySelector("h3")? Would that be more dynamic and allows future code changes more flexibility?

Reply:
yep i like this idea, will make the change